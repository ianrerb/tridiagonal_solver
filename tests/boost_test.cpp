#include <boost/math/special_functions/expint.hpp>
#include<boost/math/special_functions/gamma.hpp>
#include<iostream>

using namespace std;

int main(){ 
  cout<<boost::math::tgamma(5,.01)<<endl;
  cout<<boost::math::expint(1)<<endl;
  return 0; 
  }
