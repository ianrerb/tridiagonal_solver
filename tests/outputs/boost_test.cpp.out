24
1.89512
